library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts

```diff
@@ -28,7 +29,22 @@ export class DiscussionsComponent implements OnInit {
     this.countArr =  this.count === 2 ? [1, 2] : [1, 2, 3]
   }
 
-  handleSelectedDiscuss(discussData: any, trend: boolean): void {
-    this.router.navigateByUrl(`/app/discussion-forum/topic/${ trend ? discussData.slug : discussData.topic.slug }?page=home`)
-  }
-}
+  handleSelectedDiscuss(discussData: any, context: string | boolean): void {
+    console.log(context)
+    const subType = context && context === 'my-discussions' ? 'my-discussions' : 'trending-discussions';
+  
+    this.router.navigateByUrl(`/app/discussion-forum/topic/${discussData.slug}?page=home`);
+  
+    this.eventService.raiseInteractTelemetry(
```

Comment:
Please confirm, if this piece of code come after routing

Reply:
yes this is working